src/vanilla/Model/CodeModelTS.cs

```diff
@@ -812,5 +812,36 @@ public string GenerateReadmeMdBrowserSampleCode(string emptyLine)
 
             return builder.ToString();
         }
+
+        public string GenerateClassFields(string emptyLine)
+        {
+            TSBuilder builder = new TSBuilder();
+
+            foreach (Property property in Properties.Where(property => ShouldGenerateProperty(property.Name)))
+            {
+                builder.Line(emptyLine);
+                builder.Property(property.Name, property.ModelType.TSType(false), property.IsRequired);
```

Comment:
To keep prior behavior, please don't provided a value for `required` here.
```suggestion
                builder.Property(property.Name, property.ModelType.TSType(false));
```